index.md

```diff
@@ -332,7 +334,7 @@ Have a short `README` file explaining the project's purpose in an easy-to-find p
 This file is often the first thing users of your project will look at,
 so make it explicit already here that you welcome contributors and point them to the ways to help out.
 
-The `README` file (which may be called that, or `README.txt`, or `README.html`)
+The `README` file (which may be called that, or `README.md`, or `README.html`)
```

Comment:
I have never come across `.html` for README files, but if that is common it's OK to keep. However, I suggest keeping `.txt` as well as adding `.md`